src/.nuget/packages.builds

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <!-- This property must be set to the same value as $(PackageOutputPath) for the nuspecs and nupkgs to be binplaced to the intended location. -->
     <OutputPath>$(PackageOutputPath)</OutputPath>
+    <PackageIndexFile>$(MSBuildThisFileDirectory)packageIndex.json</PackageIndexFile>
```

Comment:
You will need to have that set in a props file that gets included in all your package projects and you need to put it into an item group called PackageIndex.  See https://github.com/dotnet/wcf/blob/master/pkg/baseline/baseline.props#L20 as an example